### DISCUSSION TITLE
Issue creation - label/milestone buttons unresponsive when comment focused

### DISCUSSION THREAD

**Discussion by haypho** [community/community#146208] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.187.0 (147865513)
Device: iPhone 14 Pro (iOS 18.1.1)
</details>

When creating an issue on mobile, the quick actions (e.g., labels, milestones, assignees, etc.) are unresponsive when the comment input is focused.

Repro Steps:
1. Navigate to a repository on GH mobile
2. Create a new issue for the repo
3. Type a title
4. Type a comment/description for the issue
5. With the comment input still focused, attempt to tap a quick action
6. Notice it is not responsive
7. Focus the title input
8. Tap a quick action
9. Notice the action flow now opens

**Comment by HenSquared** [comment]
Hey thanks @haypho - I created an internal issue to track this. 

These repro steps are helpful
